packages/devextreme/js/__internal/ui/chat/messagebox.ts

```diff
@@ -7,26 +8,35 @@ import type { Properties as DOMComponentProperties } from '@ts/core/widget/dom_c
 import DOMComponent from '@ts/core/widget/dom_component';
 import type { OptionChanged } from '@ts/core/widget/types';
 
-import type { EnterKeyEvent } from '../../../ui/text_area';
+import type { EnterKeyEvent, InputEvent } from '../../../ui/text_area';
 import type dxTextArea from '../../../ui/text_area';
 import TextArea from '../m_text_area';
 
 const CHAT_MESSAGEBOX_CLASS = 'dx-chat-messagebox';
 const CHAT_MESSAGEBOX_TEXTAREA_CLASS = 'dx-chat-messagebox-textarea';
 const CHAT_MESSAGEBOX_BUTTON_CLASS = 'dx-chat-messagebox-button';
 
+export const TYPING_START_DELAY = 1500;
```

Comment:
Did we approve these delay values for 'typingstart', 'typingend' with PMs?

Reply:
I think we can discuss it on testing later

Reply:
We decided to drop the throttle functionality [30f954d8b2d174191c45c26c9ef35cce6e9da515]